tests/generic_types/duplicate_type_specialization.c3

```diff
@@ -7,4 +7,4 @@ typedef MyType<u8> : int
 /// File *.c3', line 3, in 'typedef'
 /// Error: multiple definitions of type 'MyType<u8>':
 /// - typedef MyType<u8> : bool      (File '*.c3', line 3)
-/// - typedef MyType<u8> : bool      (File '*.c3', line 4)
+/// - typedef MyType<u8> : int       (File '*.c3', line 4)
```

Comment:
Oh damn... Can't believe I missed that

Reply:
:cry: me too. Noticing this would have saved a lot of debugging pain